python/django_bolt/testing/client.py

```
@@ -347,6 +420,7 @@ def __init__(
         read_django_settings: bool = True,
         use_http_layer: bool = True,  # Ignored - kept for backward compatibility
         static_files_config: dict | None = None,
+        share_db_connection: bool = True,
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)